heron/tmaster/src/cpp/manager/tmaster.cpp

```diff
@@ -364,7 +364,10 @@ proto::system::Status* TMaster::RegisterStMgr(
     // First check to see if that other guy has timed out
     if (!stmgrs_[stmgr_id]->TimedOut()) {
       // we reject the new guy
-      LOG(ERROR) << "Another stmgr exists with the same id and it hasn't timed out" << std::endl;
+      LOG(ERROR) << "Another stmgr exists at "
```

Comment:
@maosongfu It is right before this log, I don't know why it is hard for you to find it... At least I don't rely on one-line log for debugging, I read its context too.